.github/workflows/ci-llzk.yml

```diff
@@ -27,5 +27,27 @@ jobs:
         with:
           name: veridise-public
           authToken: '${{ secrets.CACHIX_PUBLIC_TOKEN }}'
-      - name: Build and test ${{ matrix.derivation }}
```

Comment:
not really related to the clippy stuff but `${{ matrix.derivation }}` is meaningless/empty here b/c the job is not defined with a matrix